lib/chef/knife/bmcs_server_show.rb

```diff
@@ -4,6 +4,21 @@
 require 'chef/knife/bmcs_common_options'
 require 'chef/knife/bmcs_helper'
 
+# Methods to extend the instance model
+module ServerDetails
```

Comment:
Ideally we should show all this same info during server create as well. Can this be moved to bmcs_helper.rb?